src/app/activity/bounty/Submit.ts

```diff
@@ -120,20 +138,51 @@ const writeDbHandler = async (request: SubmitRequest, submittedByUser: GuildMemb
     }
 }
 
-export const submitBountyMessage = async (message: Message, submittedByUser: GuildMember): Promise<any> => {
-	Log.debug('fetching bounty message for submit')
+// Remove message from location found. Replace with new message with correct actions in correct location
+export const submitBountyMessage = async (submittedBounty: BountyCollection, message: Message, submittedByUser: GuildMember, createdByUser: GuildMember): Promise<any> => {
+	Log.debug('fetching bounty message for submit');
 
-	const embedMessage: MessageEmbed = message.embeds[0];
+	let embedMessage: MessageEmbed = new MessageEmbed(message.embeds[0]);
+	await message.delete();
 	embedMessage.fields[BountyEmbedFields.status].value = BountyStatus.in_review;
 	embedMessage.setColor('#d39e00');
 	embedMessage.addField('Submitted by', submittedByUser.user.tag, true);
-	embedMessage.setFooter('✅ - complete |  🆘 - help');
-	await message.edit({ embeds: [embedMessage] });
-	await addSubmitReactions(message);
+	embedMessage.setFooter({text: '✅ - complete |  🆘 - help'});
+	const claimantMessage: Message = await submittedByUser.send({ embeds: [embedMessage] });
+	await addSubmitReactions(claimantMessage);
+	const creatorMessage: Message = await createdByUser.send({ embeds: [embedMessage] });
+	await addSubmitReactions(creatorMessage);
```

Comment:
Should the `claimantMesssage` also have reactions? What can the user do with those reactions?